backend/routes/group.js

```diff
@@ -466,19 +466,14 @@ router.post("/revision/:rid/groups/", middlewares.issue.canCreate, createGroup);
  */
 router.delete("/groups/", middlewares.issue.canCreate, deleteGroups);
 
-const getDbColOptions = function (req) {
-	return { account: req.params.account, model: req.params.model, logger: req[C.REQ_REPO].logger };
-};
-
 function listGroups(req, res, next) {
-
-	const dbCol = getDbColOptions(req);
 	const place = utils.APIInfo(req);
+	const { account, model } = req.params;
+	const rid = req.params.rid ? req.params.rid : null;
```

Comment:
does it have to be null? if we can use undefined then we can just merge it in wit L471